CI/Bitucket/README.md

```diff
@@ -0,0 +1,73 @@
+# Requirements
+
+* Repository Bitbucket
+
+ ![Create repository](images/create-repository.png)
+ 
+* Application configured to deploy with serverless brefphp
+
+# Step by step
+
+## Bitbucket Configuration
+
+* Go to the "Settings -> Pipelines -> Settings" menu and enable Pipeline:
+![Enable Pipeline](images/enable-pipeline.png)
+ 
+* Access the menu "Settings -> Pipelines -> Deployments"
+    - Select the environment you want to deploy (Staging or Production), in this example I will use Staging
+    - Add the variables
+        * AWS_CREDENTIAL_KEY: Access Key ID
+        * AWS_CREDENTIAL_SECRET: Secret Key
+  
+![Set Environments](images/environment-variables.png)
+
+## Project configuration
+
+* In the root directory of your project create the file `bitbucket-pipelines.yml`:
+```
+pipelines:
+  branches:
+    master: # Whenever there is a push on the master branch
+      - step: # Step 1: Using a docker php image to install composer and download your project dependencies
+          name: Composer install
+          image: php:7.3
```

Comment:
I think it would be better to run using the official Bref docker images. Else, any cache built with other images could break when run on Lambda.

For example, some pieces of Symfony or Laravel hardcode the application's path in the cache. So here it will be different in production.

Reply:
I tried to use the bref images but couldn't.
bref/dev-env the workdir is / so the app gets along with the system files
bref/php-73-fpm-dev doesn't find composer and serverless

Bitbucket does not allow me to change workdir, if you can change the dev-env image to set a workdir I believe it solves, for now I will keep the example with php and node docker images

Reply:
Understood, thank you for the details!